modules/migration-configuring-gcp.adoc

```diff
@@ -21,6 +21,7 @@ You can configure a Google Cloud Provider (GCP) storage bucket as a replication
 
 . Run `gsutil init` to log in:
 +
+[source,terminal]
 ----
 $ gsutil init
 Welcome! This command will take you through the configuration of gcloud.
```

Comment:
This output should be detached from the command and inc the .Example output title and terminal metadata ie:
[source,terminal]
----
$ gsutil init
----
+
.Example output
[source,terminal]
----
Welcome! This ... etc